From 588c60d0c3d11e79d19860fa62b03c935658d13a Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sun, 13 Jan 2013 14:36:48 +0100
Subject: [PATCH qtbase] eglconvenience: add egl to CONFIG to get correct include
 paths

The eglconvenience code includes <EGL/egl.h>. Therefore, it should get
the appropriate EGL-specific include paths from QMAKE_INCDIR_EGL,
otherwise the build might if the EGL library has its headers in
special locations. In order to achieve this, we simply add the "egl"
feature to the list of features imported by eglconvenience.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 .../eglconvenience/eglconvenience.pri              |    1 +
 1 file changed, 1 insertion(+)

diff --git a/src/platformsupport/eglconvenience/eglconvenience.pri b/src/platformsupport/eglconvenience/eglconvenience.pri
index 188eb1c..8996cea 100644
--- a/src/platformsupport/eglconvenience/eglconvenience.pri
+++ b/src/platformsupport/eglconvenience/eglconvenience.pri
@@ -5,6 +5,7 @@ contains(QT_CONFIG,egl) {
     SOURCES += \
         $$PWD/qeglconvenience.cpp \
         $$PWD/qeglplatformcontext.cpp
+    CONFIG += egl
 
     contains(QT_CONFIG,xlib) {
         HEADERS += \
-- 
1.7.9.5

